packages/core/content-manager/admin/src/pages/EditView/components/DocumentActions.tsx

```diff
@@ -293,8 +293,17 @@ const DocumentActionsMenu = ({
               key={action.id}
             >
               <Flex justifyContent="space-between" gap={4}>
-                <Flex color={convertActionVariantToColor(action.variant)} gap={2} tag="span">
-                  <Box tag="span" color={convertActionVariantToIconColor(action.variant)}>
+                <Flex
+                  color={!action.disabled ? convertActionVariantToColor(action.variant) : 'inherit'}
+                  gap={2}
+                  tag="span"
+                >
+                  <Box
+                    tag="span"
+                    color={
+                      !action.disabled ? convertActionVariantToColor(action.variant) : 'inherit'
+                    }
```

Comment:
could you check if just "color: inherit" would work here? (or even not setting color at all). We could maybe avoid having to write the color logic twice

Reply:
I can't set the color to undefined if it is that your comment, I tried if inherit work and it works because it inherit color from the variant of the wrapper

Reply:
if inherit works here that means you only need to call convertActionVariantToColor(action.variant) once right? (in the parent Flex)